AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -88,6 +94,14 @@ - (instancetype)initWithLayoutElement:(id<ASLayoutElement>)layoutElement
 
     _sublayouts = sublayouts != nil ? [sublayouts copy] : @[];
     _flattened = NO;
+    
+    // Add sublayouts layout elements to an internal array to retain it while the layout lives
+    if (sublayouts.count > 0) {
```

Comment:
Another (better?) way is to trigger strong refences gathering here: https://github.com/facebook/AsyncDisplayKit/blob/master/AsyncDisplayKit/Layout/ASLayout.mm#L174